rapids/internal/launch-dask.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# This script creates systemd configuration for Dask.
+
+set -euxo pipefail
+
+echo "Installing Dask service..."
+
+readonly ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+readonly MASTER=$(/usr/share/google/get_metadata_value attributes/dataproc-master)
+readonly RUN_CUDA_WORKER_ON_MASTER=$(/usr/share/google/get_metadata_value attributes/run-cuda-worker-on-master || echo -n 'true')
+readonly DASK_LAUNCHER='/usr/local/bin/launch-dask.sh'
+readonly INIT_SCRIPT='/usr/lib/systemd/system/dask-cluster.service'
+# For Anaconda Dataproc component
+readonly PREFIX='/opt/conda/anaconda/envs/RAPIDS/bin'
+# For miniconda dataproc init-action
+#readonly PREFIX='/opt/conda/bin'
+
+cat << EOF > "${DASK_LAUNCHER}"
+#!/bin/bash
+if [[ "${ROLE}" == 'Master' ]]; then
```

Comment:
I would install 2 different files based on the role instead of if-else inside the script.

i.e
```
if [[ "${ROLE}" == 'Master' ]]; then
  cat << EOF > "${DASK_LAUNCHER}"
# code for master
EOF
else
  cat << EOF > "${DASK_LAUNCHER}"
# code for worker
EOF
fi

Reply:
Done